### ISSUE TITLE
[Debian] ssl: got an unexpected keyword argument 'family'

### ISSUE THREAD

**Issue by bmwiedemann** [openSUSE/obs-build#341]
I'm using osc from
http://download.opensuse.org/repositories/openSUSE:/Tools/Debian_8.0/
and got a backtrace ending in

```
  File "/usr/lib/python2.7/dist-packages/osc/oscssl.py", line 270, in connect       connected = self._connect(addrinfo[0])
  File "/usr/lib/python2.7/dist-packages/osc/oscssl.py", line 253, in _connect
    self.sock = SSL.Connection(self.ssl_ctx, family=family)
TypeError: __init__() got an unexpected keyword argument 'family'
```

When dropping the family=family part, it works again, so maybe this is missing some runtime version requirement.
System has python-m2crypto 0.21.1-3

**Comment by marcus-h** [comment]
On 2017-03-12 13:46:31 -0700, Bernhard M. Wiedemann wrote:
>   File "/usr/lib/python2.7/dist-packages/osc/oscssl.py", line 270, in connect       connected = self._connect(addrinfo[0])
>   File "/usr/lib/python2.7/dist-packages/osc/oscssl.py", line 253, in _connect
>     self.sock = SSL.Connection(self.ssl_ctx, family=family)
> TypeError: __init__() got an unexpected keyword argument 'family'
> 
This is fixed in osc commit ff19a5f. See also the osc issue
#274 ("osc 0.157 Exception AttributeError: "Connection instance
has no attribute 'ssl_close_flag'"").

**Comment by bmwiedemann** [comment]
I just noticed, that I filed this on the wrong project (late in the evening is not the best time).
So this is just a dup of https://github.com/openSUSE/osc/issues/274
